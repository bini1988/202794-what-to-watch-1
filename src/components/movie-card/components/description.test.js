import React from "react";
import renderer from "react-test-renderer";
import Description from "./description";
import card from "../../../mocks/movie-card";

it(`Description correctly renders default markup`, () => {
  const tree = renderer.create(
      <Description card={card}>
        {`Childs elements`}
      </Description>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
