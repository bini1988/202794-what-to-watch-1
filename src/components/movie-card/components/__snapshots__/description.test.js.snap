// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description correctly renders default markup 1`] = `
<div
  className="movie-card__desc"
>
  <h2
    className="movie-card__title"
  >
    War of the Worlds
  </h2>
  <p
    className="movie-card__meta"
  >
    <span
      className="movie-card__genre"
    >
      Adventure
    </span>
    <span
      className="movie-card__year"
    >
      2005
    </span>
  </p>
  <div
    className="movie-card__buttons"
  >
    Childs elements
  </div>
</div>
`;
